models/lti/responsesRepository.js

```diff
@@ -54,7 +54,6 @@ const responsesRepository = () => {
   const upsert = (formResponse) => getByLti(formResponse.lti)
     .then(existingResponse => {
       if (existingResponse) {
-        formResponse._id = existingResponse._id;
```

Comment:
If there's response we're updating then we already have it's _id. If there's a case where we receive data without _id then it would be better if we'd build out new response with proper _id and updated data, and send it to the update method.
Mutating _id is bad practice either way.